static/src/js/components/OrderStatus/__tests__/OrderStatus.test.jsx

```diff
@@ -30,7 +34,46 @@ beforeEach(() => {
 })
 
 describe('OrderStatus component', () => {
-  test('renders itself correctly', () => {
+  test('renders itself correclty upon page load', () => {
+    setup({
+      overrideProps: initalizedRetrievalStatusProps
+    })
+
+    expect(screen.getByText('https://search.earthdata.nasa.gov/downloads/7')).toBeInTheDocument()
+    expect(Skeleton).toHaveBeenCalledTimes(1)
```

Comment:
On any tests where the data has been loaded, add in an assertion to ensure the Skeleton is not being rendered
```
expect(Skeleton).toHaveBeenCalledTimes(0)
```